src/condition_codes.rs

```diff
@@ -1,9 +1,9 @@
 #[derive(Debug, Default)]
 pub struct ConditionCodes {
+    pub carry: bool,
     pub zero: bool,
     pub sign: bool,
     pub parity: bool,
-    pub carry: bool,
     pub aux_carry: bool,
```

Comment:
This is just me being OCD to match the order of the bits being affected in the manual and cpu.rs